qiskit/transpiler/runningpassmanager.py

```diff
@@ -75,12 +75,20 @@ def append(self, passes, **flow_controller_conditions):
         Raises:
             TranspilerError: if a pass in passes is not a proper pass.
         """
-        flow_controller_conditions = self._normalize_flow_controller(flow_controller_conditions)
-
-        self.working_list.append(
-            FlowController.controller_factory(passes,
-                                              self.passmanager_options,
-                                              **flow_controller_conditions))
+        if isinstance(passes, ConditionalController):
+            passes.condition = partial(passes.condition, self.fenced_property_set)
```

Comment:
Can you add a comment on the purpose of the partial call? Also, can you update the docstring?

Reply:
yeah... that currying is one of the thing that I would like to get rid off in a pass manager refactor. Comment added in 6ed202d6d

docstring updated in 1f83c01dd